packages/dev/core/src/Collisions/gpuPicker.ts

```diff
@@ -90,6 +90,7 @@ export class GPUPicker {
     /**
      * Set the list of meshes to pick from
      * Set that value to null to clear the list (and avoid leaks)
+     * If set to null, the array passed will be cleared

```

Comment:
This just repeats the last sentence. Would be better to have something like:

```suggestion
     * When the module is disposed the list provided will be cleared.
```